### ISSUE TITLE
Zooming with smoothness via mouse scroll button is not smooth

### ISSUE THREAD

**Issue by 6AKU** [beyond-all-reason/Beyond-All-Reason#1534]
1. Switch to Overhead (TA) camera (Ctrl+F2).
2. Apply /option camerasmoothness 2.
3. Start moving camera and zooming via middle mouse button.

P.S Zooming via page up and page down is smooth.

https://youtu.be/wUoKo2pNv7M

**Comment by icexuick** [comment]
Correct. This should be fixed, or default or F2 should be merged with F3 imo.

**Comment by 6AKU66** [comment]
Not fixed.

**Comment by icexuick** [comment]
Really? for me Spring Cam is super smooth (zooming with scrollwheel + scrolling/panning.

**Comment by 6AKU66** [comment]
Honestly I already don't remember details. Maybe I was wrong about spring camera.

**Comment by sprunk** [comment]
There were some engine-side camera fixes since then.
